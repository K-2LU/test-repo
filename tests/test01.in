100
2120 9313 7603 8358 540 9168 5610 4023 9278 7740 8995 3116 8325 2481 8175 6635 6566 5298 8463 8289 6209 2109 5083 8697 3621 4900 4692 2166 1336 9455 765 3456 5121 4720 1814 5661 3888 3776 6036 9518 7868 1384 2634 6194 217 809 9181 6783 2459 7645 1425 8669 6106 2860 3718 9727 7760 8411 8245 5448 4218 5362 5256 9339 82 3423 1352 322 7199 3741 9840 5068 5125 8826 1262 1694 5988 6795 4829 8447 4440 6254 3468 546 9115 7187 6625 3227 5598 1223 5028 6168 6585 284 5508 3020 3707 6860 3342 7259 