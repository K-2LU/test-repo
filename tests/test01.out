18 95 74 83 5 92 55 37 94 76 90 25 82 21 79 67 64 50 86 81 62 17 46 88 31 43 40 19 11 97 7 29 47 41 16 56 36 35 58 98 78 13 22 61 2 8 93 68 20 75 14 87 59 23 33 99 77 84 80 52 38 51 49 96 1 28 12 4 72 34 100 45 48 89 10 15 57 69 42 85 39 63 30 6 91 71 66 26 54 9 44 60 65 3 53 24 32 70 27 73